core/state_transition.go

```diff
@@ -650,6 +665,9 @@ func (st *stateTransition) innerExecute() (*ExecutionResult, error) {
 				}
 				st.state.AddBalance(params.OptimismL1FeeRecipient, amtU256, tracing.BalanceIncreaseRewardTransactionFee)
 			}
+			if worstOperatorCost != nil {
+				st.state.AddBalance(params.OptimismOperatorFeeRecipient, worstOperatorCost, tracing.BalanceIncreaseRewardTransactionFee)
```

Comment:
Why are we rewarding the worst-case operator cost to the operator fee recipient? Shouldn't this be the actual cost, referencing `gasUsed`? (I think we chatted about using `gasLimit` vs. `gasUsed` yesterday in the upgrade call, but not privy to the details).

Following the standard set by the other fee checks, we want to:
1. Check that the account has balance to cover the worst case. This looks good on L289.
2. Reward only the actual operator fee, which gets computed w/ actual gas used. (See how the base fee check / reward is computed)

Reply:
This has been handled